### DISCUSSION TITLE
We lost our org, please help

### DISCUSSION THREAD

**Discussion by robb-j** [community/community#143384] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

We lost our GitHub organisation two weeks ago, is there any way someone can look at our GitHub support ticket? (3058587) ((I'm starting to get desperate))

**Comment by M-tech-cmd** [comment]
Follow Up on the Ticket: Make sure to follow up on your support ticket (3058587) through the GitHub Support portal. Sometimes, additional follow-ups can help prioritize your request.

Contact GitHub Support via Social Media: Reach out to GitHub Support on Twitter or other social media platforms. Publicly mentioning your issue can sometimes expedite the process.

Community Forums: Engage with the GitHub Community forums where you posted your issue. Other users or GitHub staff might provide additional insights or assistance.
Email Support: If you haven’t already, try emailing GitHub Support directly with all relevant details and your ticket number.

I hope this helps, and your issue gets resolved soon!
